code/modules/mob/living/carbon/xenomorph/mutators/strains/sentinel/toxic.dm

```diff
@@ -0,0 +1,52 @@
+/datum/xeno_mutator/toxic
+	name = "STRAIN: Sentinel - Toxic"
+	description = "all of your nuerotoxin is replaced with cytotoxins"
+	flavor_description = "Ssssss I am a snek"
```

Comment:
This is shown to players so needs to be better

Reply:
Updated description and flavor description.